test/cache.cpp

```diff
@@ -30,9 +30,11 @@
 
 #include <miopen/md5.hpp>
 #include "test.hpp"
+#if MIOPEN_ENABLE_SQLITE && MIOPEN_ENABLE_SQLITE_KERN_CACHE
```

Comment:
[Recommendation] MIOPEN_ENABLE_SQLITE_KERN_CACHE cannot be On when MIOPEN_ENABLE_SQLITE=Off
```suggestion
#if MIOPEN_ENABLE_SQLITE_KERN_CACHE
```
everywhere in this file

Reply:
resolved